sw/device/lib/crypto/BUILD

```diff
@@ -4,15 +4,12 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("//rules:opentitan.bzl", "OPENTITAN_CPU", "OPENTITAN_PLATFORM", "opentitan_binary")
+load("//rules/opentitan:defs.bzl", "OPENTITAN_CPU", "OPENTITAN_PLATFORM")
```

Comment:
We can probably remove this, unless it needs to be added to the new `ot_static_library` target.